packages/main/test/sap/ui/webcomponents/main/pages/Popover2.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
```

Comment:
I think that we don't need this file anymore. If you agree, delete it

Reply:
deleted, thanks